src/api.js

```diff
@@ -30,6 +30,7 @@ let api = new API({
     'This document describes the API end-point for publishing the pulse',
     'message. This is mainly intended to be used by tools.',
   ].join('\n'),
+  name:'purgecache',
```

Comment:
I think it might need a dash...

Reply:
@jonasfj i dont think it is possible since the name can only be a-z0-9 according to the pattern.. Should I change it into cachepurger instead?

Reply:
https://github.com/taskcluster/taskcluster-lib-api/blob/master/src/api.js#L650 allows a dash in characters after the first.

Reply:
@djmitche oh sorry I didnt read the regex carefully.. I updated the name